src/static/js/modules/per-page-js-loader.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var jsManifest = require( '../config/js-manifest-config.js' );
+
+/**
+ * Load a page-specific JavaScript dynamically into the header
+ * if the page has a page-specific script available.
+ */
+function init() {
+  var jsLoader = require( '../modules/util/js-loader' );
+  var rawJson = document.getElementById( 'page-javascript' ).innerHTML;
```

Comment:
So the bundles themselves are created in advance, but do you mean the lookup for whether they exist or not? Because, yes, that should happen on the server and I have a TODO [here](https://github.com/cfpb/cfgov-refresh/pull/968/files#diff-a2823c2782d047ce8761801fa918298dR198) for that. I couldn't figure out how to do it with our existing available jinja commands. Ideally it would do something like inject the per-page JS URL to load, and then the per-page-js-loader would load that URL if it was present on the page and do nothing if it wasn't. Let me know if you have any ideas!